### PR TITLE
Update/local demo

### PR DESCRIPTION
**Pull request by ccali11**
@shanejearley @DemogorGod - Operators are adding to user's table as expected. See video:

https://github.com/consensusnetworks/casimir/assets/15663412/05e52279-7054-4c7a-b9f0-605725ed9217

@DemogorGod - There might be some UI/UX bugs to work out. One example (that I tagged you by github handle in the code) is making sure to wait for successful registration before showing the new operator on the table. Can help show you how to test that.

### CODE REVIEW

File: apps/web/src/composables/contracts.ts [hunk 1]
```diff
@@ -426,6 +164,165 @@ export default function useContracts() {
...
+                    active,
+                    collateral: ethers.utils.formatEther(collateral),
+                    poolCount: poolCount.toNumber(),
+                    url,
+                    resharing,
+                    pools
+                })
+            }
+        }
+        _setOperators(ssvOperators)
```
Comment (shanejearley) [inline apps/web/src/composables/contracts.ts:268]: What if we only push non-Casimir-registered operators to unregisteredOperators and Casimir-registered operators to registeredOperators and join the lists if needed (but removes some other diffing logic elsewhere to have them separate).

**Review by shanejearley — APPROVED** [review]
No need to make all changes, some are just thoughts. @ccali11

File: apps/web/src/composables/utilities.ts [hunk 2]
```diff
@@ -0,0 +1,94 @@
+import { ref } from 'vue'
+import * as XLSX from 'xlsx'
+
+export default function useUtilities() {
```
[the remark this answers is not in the document]
Comment (ccali11) [reply apps/web/src/composables/utilities.ts:4]: what?

File: apps/web/src/composables/utilities.ts [hunk 3]
```diff
@@ -0,0 +1,94 @@
+import { ref } from 'vue'
```
[the remark this answers is not in the document]
Comment (ccali11) [reply apps/web/src/composables/utilities.ts:1]: nope!

File: apps/web/src/composables/contracts.ts (hunk 1 above)
Comment (ccali11) [reply apps/web/src/composables/contracts.ts:268]: 💡

File: apps/web/src/composables/utilities.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (ccali11) [reply apps/web/src/composables/utilities.ts]: done

**Pull request merged**